koku/masu/test/database/helpers.py

```diff
@@ -141,105 +138,92 @@ def create_ocp_report_period(self, provider_uuid, period_date=None, cluster_id=N
             data["namespace"] = namespace
         if node:
             data["node"] = node
-        with OCPReportDBAccessor(self.schema) as accessor:
-            return accessor.create_db_object(table_name, data)
+        if null_cpu_usage:
+            data["pod_usage_cpu_core_seconds"] = None
+        with schema_context(self.schema):
+            return baker.make(
+                model, report_period_id=report_period.id, report_id=report.id, **data, _fill_optional=True
+            )
 
     def create_ocp_storage_line_item(self, report_period, report, pod=None, namespace=None):
         """Create an OCP storage line item database object for test."""
         table_name = OCP_REPORT_TABLE_MAP["storage_line_item"]
-        data = self.create_columns_for_table(table_name)
-
-        for key in data:
-            if "bytes" in key or "byte" in key:
-                data[key] = data[key] * Decimal(pow(2, 30)) * 5
-
-        data["report_period_id"] = report_period.id
-        data["report_id"] = report.id
+        model = get_model(table_name)
+        data = {
+            "persistentvolumeclaim_capacity_bytes": self.fake.pydecimal(left_digits=5, right_digits=5),
+            "persistentvolumeclaim_capacity_byte_seconds": self.fake.pydecimal(left_digits=5, right_digits=5),
+            "volume_request_storage_byte_seconds": self.fake.pydecimal(left_digits=5, right_digits=5),
+            "persistentvolumeclaim_usage_byte_seconds": self.fake.pydecimal(left_digits=5, right_digits=5),
+        }
         if pod:
             data["pod"] = pod
         if namespace:
             data["namespace"] = namespace
-        with OCPReportDBAccessor(self.schema) as accessor:
-            return accessor.create_db_object(table_name, data)
+        with schema_context(self.schema):
+            return baker.make(
+                model, report_period_id=report_period.id, report_id=report.id, **data, _fill_optional=True
+            )
 
-    def create_ocp_node_label_line_item(self, report_period, report, node=None, node_labels=None):
+    def create_ocp_node_label_line_item(self, report_period, report, node=None):
         """Create an OCP node label line item database object for test."""
         table_name = OCP_REPORT_TABLE_MAP["node_label_line_item"]
-        data = self.create_columns_for_table(table_name)
-        data["report_period_id"] = report_period.id
-        data["report_id"] = report.id
+        model = get_model(table_name)
+        data = {}
         if node:
             data["node"] = node
-        with OCPReportDBAccessor(self.schema) as accessor:
-            return accessor.create_db_object(table_name, data)
+        with schema_context(self.schema):
+            return baker.make(
+                model, report_period_id=report_period.id, report_id=report.id, **data, _fill_optional=True
+            )
 
-    def create_columns_for_table(self, table):
+    def create_columns_for_table_with_bakery(self, table, data={}):
```

Comment:
The whole reason I am updating this file is because of this `create_columns_for_table` function. I _think_ the update of `faker` was causing a lot of test failures (something related to Django and Decimals).

So I re-wrote this particular function and converted most functions in this file to use model_bakery. The bakery is a better solution for generating fake data for a Django model because it complete fills the model fields (e.g. a Charfield with a max length of 256 characters is going to get an entry with 256 characters.)